app/chat/management/commands/sync_users_to_chat.py

```diff
@@ -0,0 +1,79 @@
+'''
+    Copyright (C) 2018 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from dashboard.models import Profile
+from chat.tasks import create_user
+import logging
+from celery import group
+
+logger = logging.getLogger(__name__)
+
+from marketing.utils import should_suppress_notification_email
+
+
+class Command(BaseCommand):
+    help = "create users to Gitcoin chat, creates the user if it doesn't exist"
+
+    def handle(self, *args, **options):
+        try:
+
+            users = Profile.objects.filter(user__is_active=True).prefetch_related('user')
```

Comment:
lets call this `profiles` instead of `users` - I think its very confusing to conflate them since they are two separate and specific models